bower.json

```diff
@@ -24,7 +24,7 @@
     "bootstrap-stylus": "~4.0.2",
     "braintree-angular": "~1.2.6",
     "card": "1.0.0",
-    "font-awesome": "~4.3.0",
+    "font-awesome": "~4.5.0",
```

Comment:
Could you please separate upgrade to font awesome 4.5.0 and code for the issue ?
two PR or two commits